payments/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+
+class ProcessedTransaction(models.Model):
```

Comment:
Just thinking if we do store transaction, do we want to store more metadata? Such as bank account from which we received payment, amount received, other useful info that can be used on invoice?